web/src/rest.js

```diff
@@ -1,14 +1,19 @@
 
+import axios from 'axios';
 import { RestClient } from '@girder/components/src';
 
 const apiRoot = process.env.VUE_APP_API_ROOT || 'http://localhost:8080/api/v1';
+const publishApiRoot = process.env.PUBLISH_APP_API_ROOT || 'http://localhost:8000/api';
```

Comment:
I think it'd be better to to define a typical development default of the publish API root environment variable in `.env.development`, and omit the default value here (likewise with `apiRoot`). For production builds, I don't think we want this fallback default, as we want the app to fail obviously if the environment was not configured.